regression-tests.dnsdist/test_TCPLimits.py

```diff
@@ -106,11 +106,11 @@ def testTCPDuration(self):
         conn.send(struct.pack("!H", 65535))
 
         count = 0
-        while count < (self._maxTCPConnDuration * 2):
+        while count < (self._maxTCPConnDuration * 20):
             try:
                 # sleeping for only one second keeps us below the
                 # idle timeout (setTCPRecvTimeout())
-                time.sleep(1)
+                time.sleep(0.5)
```

Comment:
Didn't you use `0.1` in the initial version of this change? This doesn't match the other changes and the tests rightly fail.

Reply:
Huh, yes, looks like I committed some random experiment. Pushed, will squash things a bit if CI is green.